webpack.config.js

```diff
@@ -4,7 +4,8 @@ const path = require( 'path' );
 
 module.exports = {
 	entry: {
-		'./assets/js/amp-blocks-compiled': './blocks/index.js'
+		'./assets/js/amp-blocks-compiled': './blocks/index.js',
+		'./assets/js/amp-block-editor-toggle-compiled': './assets/src/amp-block-editor-toggle.js'
```

Comment:
I think these two two should be combined into a single entry point. In other words, `./blocks/index.js` could just import the module for the block editor toggle, and then there wouldn't be a need for `amp-block-editor-toggle-compiled.js`

Reply:
Good idea, I'll apply that.

Reply:
Could this instead combine `amp-block-editor-toggle-compiled.js` with any future modules, like [amp-block-validation.js](https://github.com/Automattic/amp-wp/blob/e3d036c2db60b215636a2b665414b0e599e9ad97/assets/js/amp-block-validation.js) (when that's converted to a module)?

It looks like `assets/js/amp-blocks-compiled.js` is [only enqueued if amp_is_canonical()](https://github.com/Automattic/amp-wp/blob/e3d036c2db60b215636a2b665414b0e599e9ad97/includes/admin/class-amp-editor-blocks.php#L127).

Reply:
@kienstra Yes, `amp-block-validation.js` should also be converted to a module and included in this bundle. 100%

Good point on `amp-blocks-compiled.js`. Maybe that would make sense to be a separate bundle then. Either that, or there could be an arg passed for whether the blocks should be registered. But if not, then why include the file in the first place?

Reply:
So `amp-block-validation.js` and `amp-block-editor-toggle-compiled.js` should definitely be combined.

Reply:
Thanks, combining `amp-block-validation.js` and `amp-block-editor-toggle-compiled.js` sounds good. 

If it's alright, I'll open a new issue to convert `amp-block-validation.js` to a module and compile it in the same file.

Reply:
Sounds good. 👍

Reply:
I know that @hellofromtonya is currently working on `amp-block-validation.js` so it would be good to not step on her toes 😄

Reply:
@kienstra One more thing for that issue you'll create: please add to it the need to generate source maps.

Reply:
@westonruter, @kienstra and I synced up in slack.  He'll open the issue.  Once we resolve the issues in `amp-block-validation.js`, then we can green light the task.

Reply:
Issue #1298 now tracks bundling [amp-block-editor-toggle.js](https://github.com/Automattic/amp-wp/blob/0735ed02ef01f85782d38e2a7de56e42b085c8e9/assets/src/amp-block-editor-toggle.js#L1) with [amp-block-validation.js](https://github.com/Automattic/amp-wp/blob/21ebbc69289ec46a6752df39d2170a625c70faf7/assets/js/amp-block-validation.js). It also mentions that it's probably best to wait for #1293.